plugins/gatsby-plugin-auto-index-pages/gatsby-node.js

```diff
@@ -153,7 +151,7 @@ exports.createPages = async ({ actions, graphql, reporter }, pluginOptions) => {
     if (skippedDirectories.includes(dir.path)) {
       return [visit.SKIP];
     }
-
+    // I think we want to just look at tableofcontents, not all filenodes
```

Comment:
Nitpick: can we either (a) remove this comment or (b) add more context? It feels like this might have been a note while debugging that snuck into the PR.

Reply:
thanks for catching that. was a superfluous comment